efsity/src/main/java/org/smartregister/util/FCTConstants.java

```diff
@@ -5,5 +5,12 @@
 public class FCTConstants {
   public static final Set<String> questionnaireTranslatables = Set.of("text", "display");
   public static final Set<String> configTranslatables =
-      Set.of("saveButtonText", "title", "display", "actionButtonText", "message");
+      Set.of(
+          "saveButtonText",
+          "title",
+          "display",
+          "actionButtonText",
+          "message",
+          "primaryText",
+          "secondaryText");
```

Comment:
@ndegwamartin is it possible to make this configurable by adding it to a config file or something. That we don't have to change much in future?

Reply:
I don't expect there to be too many such variables in the configs. We probably want to restrict it to a finite and small set and re-use as much. Either way I'd propose we merge as is and consider configuration based on any future similar requirements cc @lincmba 

One way we can refactor in future is to introduce some type of (global) property file for efsity, probably `.yaml` format which can have various configurable entries section, one of them being the above e.g.

```
efsity-config.yml

---
questionnaireTranslatables:
     - text
     - display

configTranslatables:
      - saveButtonText
      - title
      - display
      - actionButtonText
      - message
      - primaryText
      - secondaryText